api/v1beta1/ssp_webhook.go

```diff
@@ -65,6 +69,11 @@ func (r *SSP) ValidateCreate() error {
 		return fmt.Errorf("creation failed, the configured namespace for common templates does not exist: %v", namespaceName)
 	}
 
+	err = validatePlacement(r)
```

Comment:
For simplicity and coding size, I like the error testing in only one line:

if err = validatePlacement(r); err != nil {
	return err
}

Reply:
Done